### DISCUSSION TITLE
How to fix Bad Conformer Id

### DISCUSSION THREAD

**Discussion by kelsingo** [rdkit/rdkit#7976] [Q&A]
`CC(C)[C@@H]1CC[C@@]2([C@H]1[C@@H]3C(=C)CC[C@@H]2S3)C`
I tried to convert this string to 3D SDF, but RDKit returns Bad Conformer Id. I would be grateful if you can help me to identify the causes and suggest some options to fix this!!!

**Comment by ptosco** [comment]
@kelsingo This is because your SMILES specifies a stereochemistry which does not correspond to a feasible 3D conformation:
```
from rdkit import Chem

mol = Chem.MolFromSmiles("CC(C)[C@@H]1CC[C@@]2([C@H]1[C@@H]3C(=C)CC[C@@H]2S3)C")
mol
```
![image](https://github.com/user-attachments/assets/a9967704-ab05-4b18-ab99-de43c488e0c3)

The SMILES is requesting that the bridgehead hydrogens point at opposite directions, which means that the sulfide bridge should cross the plane of the cycloheptane ring. This would give rise to an extremely strained geometry which is unfeasible, and therefore will not be returned by RDKit.
